docs/page/configuration.md

```diff
@@ -109,6 +109,31 @@ settings.
             :name "My App"))
     ```
 
+    `dap-python` supports also the "attach" mode to attach and debug a long running script.
+    A template named "Python :: Attach to running process" is also pre-registered for this purpose.
+    ```elisp
+    (dap-register-debug-template "Python :: Attach to running process"
+      (list :type "python"
+            :request "attach"
+            :processId "${command:pickProcess}"
+            :name "Python :: Attach to running process"))
+    ```
+    The `${command:pickProcess}` configuration variable used by default to facilitate the user
+    selecting the debug process by a completion popping up window. The real `processId` can
+    always be specified using its *pid*.
+
+    **Note**: on Linux this is achieved using the [ptrace syscall](https://www.man7.org/linux/man-pages/man2/ptrace.2.html "ptrace syscall man page")
+    wrapped inside the gdb tool. Which mean below additional steps are also need to be done as well:
```

Comment:
```suggestion
    wrapped inside the GDB tool, which means that for distributions that enable YAMA (e.g. Ubuntu) some additional steps may be necessary:
```

On this paragraph: shouldn't we state this is specific to distributions that use YAMA, like Ubuntu? From this thread I could gather that Fedora doesn't enable it by default: https://www.reddit.com/r/Fedora/comments/aeu3rj/yama_lsm_ptrace_restriction_disabled_by_default/.

Reply:
agree, it's better to say it that way.